geometry_msgs/PoseStamped start_pose_ground
geometry_msgs/PoseStamped start_pose_air
---
geometry_msgs/PoseStamped next_frontier_ground
geometry_msgs/PoseStamped next_frontier_air
